.github/workflows/publish-dev.yml

```
@@ -51,13 +51,22 @@ jobs:
           node-version: 18
           registry-url: https://registry.npmjs.org/
 
+      - name: Check previous released version
```

At this time in the action, it's not actually a previous release, but rather, the current development release:

```suggestion
      - name: Check the current development version
```